x-pack/solutions/security/plugins/security_solution/public/common/utils/privileges/index.ts

```diff
@@ -6,9 +6,8 @@
  */
 
 import type { Rule } from '../../../detection_engine/rule_management/logic';
-import * as i18nActions from '../../../detections/pages/detection_engine/rules/translations';
+import * as i18nActions from '../../../detection_engine/common/translations';
```

Comment:
NIT: This file lives in a shared place (`public/common/util/privileges`) but loads stuff from inside the `detection_engine/` directory, which does not make much sense.

The `explainLackOfPermission` function, which is the one that uses these things, is imported only by files inside `detection_engine/`, so I don't understand what is this function doing here 😅 
Consider moving it to somewhere inside `detection_engine/common`

Reply:
The goal of this PR was to only move a few files from the `detections` folder into `detection_engine`. There are MANY files in the `common` folder that don't really belong in there. I don't really want to start looking at all of these and move them in this PR, this task would be pretty huge and increase even more the already big number of modified files in the PR...

One day we need to go through this `common` folder and clean it up, there are way to many files in there that are actually used in a single place and should be moved accordingly...